src/modules/Security.rb

```diff
@@ -363,6 +340,80 @@ def read_kernel_settings
         val = default_value if val.nil? || val == ""
         @Settings[key] = val
       end
+
+      log.debug "Settings (after #{__callee__}): #{@Settings}"
```

Comment:
BTW, this is another place in Yast where `__callee__` has  been already used: https://github.com/yast/yast-yast2/blob/8eb9214265fdf8afcae1531c5a0309799028d79d/library/packages/src/modules/Product.rb#L49